commands/Owoify.js

```diff
@@ -0,0 +1,55 @@
+import owoify from "owoify-js";
+import Command from "./base/Command.js";
+import { EMOTE_ID_REGEX } from "../utils/discordPatterns.js";
+
+export default class Owoify extends Command {
+  intensities = {
```

Comment:
This should be a private prop
```suggestion
  #intensities = {
```